utils/utils.js

```diff
@@ -4,31 +4,62 @@ const Discord = require("discord.js"),
 const errorLog = new Discord.WebhookClient(config.error);
 
 const utils = {
-  botSpam: function(msg) {
-    if ((msg.guild?.id == config.ldsg) && (msg.channel.id != config.channels.botspam) && (msg.channel.parentID != "363020585988653057") && (msg.channel.id != "209046676781006849")) {
+  /**
+   * If a command is run in a channel that doesn't want spam, returns #bot-lobby so results can be posted there.
+   * @param {Discord.Message} msg The Discord message to check for bot spam.
+   */
+  botSpam: function (msg) {
+    if (msg.guild?.id === config.ldsg && // Not in server
+      msg.channel.id !== config.channels.botspam && // In bot-lobby
+      msg.channel.id !== config.channels.gai && // In Gai's channel
+      msg.channel.parentID !== config.channels.moderation) { // In the moderation category
+
       msg.reply(`I've placed your results in <#${config.channels.botspam}> to keep things nice and tidy in here. Hurry before they get cold!`)
         .then(Utils.clean);
       return msg.guild.channels.cache.get(config.channels.botspam);
-    } else return msg.channel;
+    } else {
+      return msg.channel;
+    }
   },
+  /**
+   * After the given amount of time, attempts to delete the message.
+   * @param {Discord.Message} msg The message to delete.
+   * @param {number} t The length of time to wait before deletion, in milliseconds.
+   */
   clean: async function(msg, t = 20000) {
     await utils.wait(t);
     if (msg.deletable && !msg.deleted) {
       msg.delete().catch(utils.noop);
     }
     return Promise.resolve(msg);
   },
+  /**
+   * After the given amount of time, attempts to delete the interaction.
+   * @param {Discord.Interaction} interaction The interaction to delete.
+   * @param {number} t The length of time to wait before deletion, in milliseconds.
+   */
   cleanInteraction: async function(interaction, t = 2000) {
     await utils.wait(t);
     interaction.deleteReply();
   },
   Collection: Discord.Collection,
+  /**
+   * Returns a MessageEmbed with basic values preset, such as color and timestamp.
+   * @param {any} data The data object to pass to the MessageEmbed constructor. 
+   *   You can override the color and timestamp here as well.
+   */
   embed: function(data) {
     const embed = new Discord.MessageEmbed(data);
     if (!data?.color) embed.setColor(config.color);
     if (!data?.timestamp) embed.setTimestamp();
     return embed;
   },
+  /**
+   * Handles a command exception/error. Most likely called from a catch.
+   * Reports the error and lets the user know.
+   * @param {Error} error The error to report.
+   * @param {Discord.Message} message
```

Comment:
`message` can be any of a `Discord.Message`, `Discord.Interaction`, or text string.

Reply:
Error on my part below. `interaction` should be replaced by `message` in lines 83, 85, and 89 of the updated file